agent/firewall.go

```diff
@@ -42,13 +42,17 @@ import (
 )
 
 const (
-	numberOfDefaultChains = 3 // INPUT,OUTPUT,FORWARD
+	numberOfDefaultChains = 4 // INPUT,OUTPUT,FORWARD_IN,FORWARD_OUT,TENANT_VECTOR
```

Comment:
I thought so. This means the variable name is wrong then. The variable name is for humans and there is no way that "number of" would ever be considered to be something besides the number of something. When I have one item then the number of items I have is not ever zero!